cluster/kubemark/pre-existing/config-default.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Configuration for landing a Kubemark cluster on a pre-existing Kubernetes
+# cluster.
+
+# Pre-existing cluster expects a MASTER_IP
+MASTER_IP="${MASTER_IP:-}"
+MASTER_IP_AND_PORT="${MASTER_IP_AND_PORT:-$MASTER_IP:6443}"
+
+# $CONTAINER_REGISTRY/$PROJECT/kubemark
+PROJECT="${PROJECT:-kubemark}"
```

Comment:
The reason why I default ```PROJECT``` is because if I don't it will never get set: ```gcrio//kubemark```.  detect-project would normally do that, but I moved that functionality here.  But, I can live with checking it's not empty in detect-project.